specs/core/0_beacon-chain.md

```diff
@@ -311,27 +321,101 @@ Unless otherwise indicated, code appearing in `this style` is to be interpreted
 }
 ```
 
-#### `ProposalSignedData`
+#### Deposits
+
+##### `Deposit`
 
 ```python
 {
-    # Slot number
+    # Receipt Merkle branch
+    'merkle_branch': '[hash32]',
+    # Merkle tree index
+    'merkle_tree_index': 'uint64',
+    # Deposit data
+    'deposit_data': {
```

Comment:
Whoops, I didn't notice this before. Maybe move it to a new structure? (In other PRs)

Reply:
yeah, separate pr